packages/talos_messenger_actions/src/kafka/models.rs

```diff
@@ -2,12 +2,41 @@ use ahash::HashMap;
 use serde::{Deserialize, Serialize}; // 1.0.130
 use serde_json::{self};
 
+fn default_text_plain_encoding() -> String {
+    "text/plain".to_string()
```

Comment:
Are there any reasons why did you methods instead of constants?

Reply:
I actually didn't like this approach as well, but serde supports assigning custom default values only via `functions` or `methods`. 

You can refer to this [long running issue](https://github.com/serde-rs/serde/issues/368) which was opened in 2016, and still active discussions on it.  

I looked at some suggestions, and thought maybe it's an overkill for now, and just kept it simple with the function